.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to Microsite
+
+This codebase is a monorepo, set up with [Lerna](https://github.com/lerna/lerna). You should start by familiarizing yourself with `lerna`, as it operates a little bit differently than the standard Node/NPM workflow. 
+
+> Note: Commands should generally be run from the root directory of the monorepo rather than a subdirectory like `packages/microsite`.
+
+## Initial dev setup
+
+1. Install [Volta](https://volta.sh/) if you haven't already. It's our favorite tools for managing Node/NPM versions, and automatically picks up the correct Node version for this project.
+
+2. Install dependencies for the monorepo: `npm install`.
+
+3. Install and link dependencies in sub-packages: `npx lerna bootstrap`
```

Comment:
I see you’ve landed on leaving off `hoist`, is that intentional? I certainly think it’ll improve the top level dependency flow for now. But just wanted to check that this is actually the preferred flow.

Reply:
Yep, intentional! Seems to work just fine.

Reply:
It’s a whole lot less fussy! I will say that I’m certain Yarn (1) would make hoisting work better for perf and a better FS impact. I can almost guarantee PNPM would too, just haven’t tried it. But this is definitely a case where perfection is something I’d avoid solving, I’m just happy to know I’m all good to install a top level dependency without wanting to take a smoke break during the re-bootrappening